apel/common/message_schemas.py

```diff
@@ -0,0 +1,149 @@
+"""A file to store message schemas for JSON based messages"""
+
+IP_MSG_SCHEMA = {
+    "definitions": {
+
+    },
+    "$schema": "http: //json-schema.org/draft-07/schema#",
+    "$id": "http: //example.com/root.json",
+    "type": "object",
+    "title": "The Root Schema",
+    "required": [],
+    "properties": {
+        "Type": {
+            "$id": "#/properties/Type",
+            "type": "string",
+            "title": "The Type Schema",
+            "const": "APEL Public IP message"
+        },
+        "Version": {
+            "$id": "#/properties/Version",
+            "type": "string",
+            "title": "The Version Schema",
+            "enum": [
+                "v0.2"
+            ],
+
+        },
+        "UsageRecords": {
+            "$id": "#/properties/UsageRecords",
+            "type": "array",
+            "minItems": 1,
+            "maxItems": 1000,
+            "title": "The UsageRecords Schema",
+            "items": {
+                "$id": "#/properties/UsageRecords/items",
+                "type": "object",
+                "title": "The UsageRecord Schema",
+                "required": [
+                    "MeasurementTime",
+                    "SiteName",
+                    "CloudType",
+                    "LocalUser",
+                    "LocalGroup",
+                    "GlobalUserName",
+                    "FQAN",
+                    "IPVersion",
+                    "IPCount"
+                ],
```

Comment:
Where fields (like `LocalUser` can be `"null"` do we require them to be in the message?